bin/oref0-setup.sh

```diff
@@ -247,14 +247,22 @@ fi
 mkdir -p $HOME/src/
 if [ -d "$HOME/src/oref0/" ]; then
     echo "$HOME/src/oref0/ already exists; pulling latest"
-    (cd ~/src/oref0 && git fetch && git pull) || die "Couldn't pull latest oref0"
+    cd ~/src/oref0
+    OLD_HEAD=$(git rev-parse HEAD)
+    (git fetch && git pull) || die "Couldn't pull latest oref0"
+    NEW_HEAD=$(git rev-parse HEAD)
+    if [ "$OLD_HEAD" == "$NEW_HEAD" ];  then
+       echo "(Re)installing oref0 installation from ~/src/oref0. version=$NEW_HEAD"
+       npm run global-install
+    else 
+       echo "Not need to reinstall. version=$NEW_HEAD is still current"
+    fi
 else
-    echo -n "Cloning oref0: "
+   echo -n "Cloning oref0: "
     (cd ~/src && git clone git://github.com/openaps/oref0.git) || die "Couldn't clone oref0"
+    echo Checking oref0 installation
+    npm list -g oref0 | egrep oref0@0.3. || (echo Installing latest oref0 && sudo npm install -g oref0)
```

Comment:
If we're going to install from source any time HEAD changes, doesn't that make this part irrelevant?